quinn-proto/src/shared.rs

```diff
@@ -61,8 +59,10 @@ pub(crate) enum EndpointEventInner {
 /// Mainly useful for identifying this connection's packets on the wire with tools like Wireshark.
 #[derive(Clone, Copy, Eq, PartialEq, Ord, PartialOrd, Hash)]
 pub struct ConnectionId {
-    len: u8,
-    bytes: [u8; MAX_CID_SIZE],
+    /// length of CID
+    pub len: u8,
+    /// CID in byte array
+    pub bytes: [u8; MAX_CID_SIZE],
```

Comment:
Would it be sufficient to instead export `ConnectionId::new` and otherwise rely on the pre-existing `Deref` and `DerefMut` impls?

Reply:
Should be enough to use `ConnectionId::new`. I am reverting the above changes.

Reply:
Yes. I think it is enough to use `ConnectionId::new`. I am reverting the above changes.